app/code/community/SomethingDigital/EnterpriseIndexPerf/Model/Url/Rewrite/Redirect/Refresh.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+class SomethingDigital_EnterpriseIndexPerf_Model_Url_Rewrite_Redirect_Refresh
+    extends Enterprise_UrlRewrite_Model_Index_Action_Url_Rewrite_Redirect_Refresh
+{
+    const REWRITE_CHUNK_SIZE = 5000;
+
+    /**
+     * Clean old url rewrites records from table
+     *
+     * @return Enterprise_UrlRewrite_Model_Index_Action_Url_Rewrite_RefreshAbstract
+     */
+    protected function _cleanOldUrlRewrite()
+    {
+        $selects = $this->_prepareSelectsByRange($this->_getCleanOldUrlRewriteSelect(), 'url_rewrite_id');
+        foreach ($selects as $select) {
+            $this->_connection->query($select->deleteFromSelect('ur'));
```

Comment:
Yes, that's the same alias it deletes from in the base code - just without the chunking.